Source/ACE.Server/WorldObjects/Player_Bank.cs

```diff
@@ -438,5 +518,10 @@ public bool TransferLuminance(long Amount, string CharacterDestination)
             get => GetProperty(PropertyInt64.BankedLegendaryKeys) ?? 0;
             set { if (!value.HasValue) RemoveProperty(PropertyInt64.BankedLegendaryKeys); else SetProperty(PropertyInt64.BankedLegendaryKeys, value.Value); }
         }
+        public long? BankedPeas
```

Comment:
I don't think you need this property, since you're depositing their value to peas above, you're not setting this anywhere. We can remove it